libfreefare/freefare.c

```diff
@@ -201,6 +205,17 @@ bool freefare_selected_tag_is_present(nfc_device *device)
     return (nfc_initiator_target_is_present(device, NULL) == NFC_SUCCESS);
 }
 
+/*
+ * Set NFC operation timeout
+ */
+void
+freefare_set_tag_timeout(FreefareTag tag, int timeout)
+{
+    if (tag) {
```

Comment:
Calling `freefare_set_tag_timeout` with a NULL tag is definitely an error, no need to silently detect and ignore this: a segfault is a better indication of a problem.

The only exception to this is for `freefare_free_tag` for compatibility with `free(3)` :wink: